File: src/altair_cpp_tests/src/rclnode_test.cpp
#include "altair_cpp_tests/rclnode_test_node.hpp"



int main(int argc, char *argv[]) {
    rclcpp::init(argc, argv);
    auto node = std::make_shared<altair::RclNodeTestNode>();
    
    rclcpp::spin(node);
    rclcpp::shutdown();

    return 0;
}

File: src/altair_py_tests/altair_py_tests/rclnode_test.py
import rclpy
from .modules.rclnode_test_node import RclNodeTestNode



def main(args=None) -> None:
    rclpy.init()
    node = RclNodeTestNode()

    rclpy.spin(node)

    node.destroy_node()
    rclpy.shutdown()



if __name__ == '__main__':
    main()